Documentation/Signoffs/Mario Kart Rev 2 - Signoff - Mario Kart Bike Wireless Sensors Subsystem.md

```diff
@@ -0,0 +1,42 @@
+# Mario Kart Rev 2 - Signoff - Mario Kart Bike Wireless Sensors Subsystem 
+## Team 4 - Blake Pickett ##
+
+_Function of the Subsystem:_
+
+The function of the newly designed feature is to convert the speed and steering sensors and the microcontrollers on the Mario Kart Bike to be wireless. This new feature will mitigate safety risks to the Mario Kart Bike riders by eliminating the long wires running down the bike and around the bike stand. These long wires are a safety hazard for future customers. The wires that are rectifying and stepping down the power for the sensors on the current version of the Mario Kart Bike will be replaced with a three-AA-battery holder, complete with three AA batteries that will produce 4.5 V and 9 A current to the sensors and microcontrollers. 
+
+_Constraints:_ 
```

Comment:
Constraints don't consider speed of communication and resolution.

Reply:
Previous comment wasn't addressed. Further, I don't think you have considered how long the system will function on battery. What would be an acceptable battery life?

Reply:
For the previous comment, I had discussed that the data throughput of the Nano 33 BLE will be around 0.27 Mbps. Is this not speed of communication? Also, I'm not sure what resolution means.

Reply:
Yes, but how do you know that this is sufficient?